distribution/server-x-dist/assembly.xml

```diff
@@ -102,7 +102,7 @@
             <outputDirectory>conf</outputDirectory>
         </file>
         <file>
-            <source>target/keycloak-quarkus-server-app/META-INF/keycloak.properties</source>
+            <source>../../quarkus/runtime/src/main/resources/META-INF/keycloak.properties</source>
```

Comment:
Now, the `keycloak.properties` file is within the runtime module. The main reasons are:

* Being the default settings, it is now also loaded from our JAR.
* If a user accidentally removes this file from the distribution (e.g.: `conf/keycloak.properties`) the server is still fully functional. It should be possible to easily revert changes.
* We have the possibility to make the `conf/keycloak.properties` even simpler by just enabling what we think people should care about. For instance, we could just have there the recommended settings for "prod" profile.